pkg/activator/activator.go

```diff
@@ -168,7 +164,9 @@ func (a *activator) addAPI(obj interface{}) {
 	a.logger.Debugw("adding new api activator", zap.String("apiName", apiName))
 
 	a.activatorsMux.Lock()
-	a.apiActivators[apiName] = newAPIActivator(apiMetadata.maxQueueLength, apiMetadata.maxConcurrency)
+	if a.apiActivators[apiName] != nil {
```

Comment:
Should this be `== nil`?

Also is it possible to check this (and return) at the top of the function? e.g.

```go
if a.apiActivators[apiName] != nil {
    return
}
```